docs/TIMEOUT_WARNING.md

```diff
@@ -0,0 +1,40 @@
+# Session Timeout Warning configuration
+
+Stark provides a nice feature when it comes to session expiration handling: in case the user's session is about to end due
+to inactivity (the user is idle for some time) and you want to warn him about this, a dialog will be displayed.
+
+This is what the `SessionTimeoutWarningDialogComponent` does and also asks the user if its session should be kept alive.
```

Comment:
Let's add this in the same line to explain how we do this ;)

"This warning dialog is displayed by an NGRX Effect that triggers when a `StarkSessionActionTypes.SESSION_TIMEOUT_COUNTDOWN_START` action is dispatched (by the Session service when the user becomes idle)."